web_src/js/features/issuesutil.js

```diff
@@ -0,0 +1,1471 @@
+        $previewPanel.html(data);
+        renderMarkdownContent();
+      });
+    });
+  }
+}
+
+
+export function initIssueComments() {
+  if ($('.repository.view.issue .timeline').length === 0) return;
+
+  $('.re-request-review').on('click', function (event) {
+    const url = $(this).data('update-url');
+    const issueId = $(this).data('issue-id');
+    const id = $(this).data('id');
+    const isChecked = $(this).data('is-checked');
+
+    event.preventDefault();
+    updateIssuesMeta(
+      url,
+      '',
+      issueId,
+      id,
+      isChecked
+    ).then(reload);
+  });
+
+  $(document).on('click', (event) => {
+    const urlTarget = $(':target');
+    if (urlTarget.length === 0) return;
+
+    const urlTargetId = urlTarget.attr('id');
+    if (!urlTargetId) return;
+    if (!/^(issue|pull)(comment)?-\d+$/.test(urlTargetId)) return;
+
+    const $target = $(event.target);
+
+    if ($target.closest(`#${urlTargetId}`).length === 0) {
+      const scrollPosition = $(window).scrollTop();
+      window.location.hash = '';
+      $(window).scrollTop(scrollPosition);
+      window.history.pushState(null, null, ' ');
+    }
+  });
+}
+
+export function initLabelEdit() {
+// Create label
+  const $newLabelPanel = $('.new-label.segment');
+  $('.new-label.button').on('click', () => {
+    $newLabelPanel.show();
+  });
+  $('.new-label.segment .cancel').on('click', () => {
+    $newLabelPanel.hide();
+  });
+
+  createColorPicker($('.color-picker'));
+
+  $('.precolors .color').on('click', function () {
+    const color_hex = $(this).data('color-hex');
+    $('.color-picker').val(color_hex);
+    $('.minicolors-swatch-color').css('background-color', color_hex);
+  });
+  $('.edit-label-button').on('click', function () {
+    $('.color-picker').minicolors('value', $(this).data('color'));
+    $('#label-modal-id').val($(this).data('id'));
+    $('.edit-label .new-label-input').val($(this).data('title'));
+    $('.edit-label .new-label-desc-input').val($(this).data('description'));
+    $('.edit-label .color-picker').val($(this).data('color'));
+    $('.minicolors-swatch-color').css('background-color', $(this).data('color'));
+    $('.edit-label.modal').modal({
+      onApprove() {
+        $('.edit-label.form').trigger('submit');
+      }
+    }).modal('show');
+    return false;
+  });
+}
+
+
+export function reload() {
+  if ($('#current-card-details').length > 0) {
+    reloadIssuesActions();
+  } else {
+    window.location.reload();
+  }
+}
+
+// subscribe-repo
+// toggle_stopwatch_form"
+// cancel_stopwatch_form"
+// add_time_manual_form
+// addDependencyForm
+function initAjaxForms(formId) {
+  $(formId).submit(function(e) {
+    e.preventDefault(); // avoid to execute the actual submit of the form.
+
+    const form = $(this);
+    const url = form.attr('action');
+
+    $.ajax({
+      type: 'POST',
+      url,
+      data: form.serialize(), // serializes the form's elements.
+      success(_data) {
```

Comment:
```suggestion
      success: (_data) => {
```

Reply:
@silverwind why?

Reply:
Mostly a personal preference for this form of object members I guess.